librad/tests/propagation_basic.rs

```diff
@@ -335,7 +359,7 @@ async fn all_metadata_returns_only_local_projects() {
 /// Given that a) a peer 1 holds a given URN and b) that same peer is a seed of
 /// a peer 2, verify that requesting peer 2 for providers for said URN returns
 /// peer 1.
-#[tokio::test]
+#[tokio::test(core_threads = 4)]
```

Comment:
This one's now flaky :man_shrugging: :fried_shrimp:

Reply:
This worked on my machine with 2 threads :shrug: